keras/src/export/tfsm_layer.py

```diff
@@ -0,0 +1,139 @@
+from keras.src import backend
+from keras.src import layers
+from keras.src.api_export import keras_export
+from keras.src.export.saved_model import _list_variables_used_by_fns
+from keras.src.utils.module_utils import tensorflow as tf
+
+
+@keras_export("keras.layers.TFSMLayer")
```

Comment:
We might want to move the file to `keras/layers/` in the future. Fine to keep it here for now.